CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 * @NordSecurity/libtelio-team
 .github/workflows @Jauler @jjanowsk @packgron @tomaszklak
+nat-lab/* @gytsto
```

Comment:
If I'm reading correctly the [documentation example](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax):
```
# In this example, @doctocat owns any files in the build/logs
# directory at the root of the repository and any of its
# subdirectories.
/build/logs/ @doctocat

# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```
what you did will only make you an owner of files directly in the nat-lab directory but not for files deeper. I think you just want:
```
nat-lab/ @gysto
```

Reply:
or just
```
nat-lab @gysto
```
to match style with worklows

Reply:
alrighty, thanks for catching my bs... updated :smile: